app/views/CancelVATRegistrationPage.scala.html

```diff
@@ -27,12 +28,15 @@
     formHasErrors = form.errors.nonEmpty,
```

Comment:
I believe date inputs also use fieldsets so we made need to change that, do we want to do that in this PR or a new subtask?